client/src/components/Main.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import DeleteBtn from './DeleteBtn';
+import Videos from '../Videos';
```

Comment:
I want you to check this import statement again. When I try to run the React app in my device, it gives me the error `Module not found: Error: Can't resolve '../Videos' in '/Users/alicagatay/Documents/Dev/CodeYourFuture/West-Midlands-Class-4/Full Stack Project/Full-Stack-Project-Assessment/client/src/components'`. 

Clue: You are trying to import a module with your own computers directory, but not everyone has directory in their computers or servers.

Reply:
:+1: